code/modules/uplink/uplink_items/species.dm

```diff
@@ -5,6 +5,7 @@
 /datum/uplink_item/species_restricted
 	category = /datum/uplink_category/species
 	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
+	surplus = 0
```

Comment:
Undocumented change, reeeee

Reply:
It was ACCIDENTAL when I FORGOT to add them in ok man FINE